frontend/src/components/VSourcesTable.vue

```diff
@@ -167,37 +191,79 @@ export default defineComponent({
 @tailwind utilities;
 
 @layer components {
-  .table {
-    @apply rounded-sm border-0 border-dark-charcoal-20;
+  .source-table {
+    @apply hidden rounded-sm border-0 border-dark-charcoal-20 md:table;
   }
-  .table th,
-  .table td {
+
+  .source-table th,
+  .source-table td {
     @apply border-dark-charcoal-20;
   }
-  .table a {
+
+  .source-table a {
     @apply text-pink hover:underline;
   }
-  .table th {
+
+  .source-table th {
     @apply cursor-pointer border-t bg-dark-charcoal-10;
   }
-  .table th,
-  .table td {
+
+  .source-table th,
+  .source-table td {
     @apply border-r p-4 first:border-l;
   }
-  .table td {
+
+  .source-table td {
     @apply break-normal border-y-0;
   }
 
-  .table tr {
+  .source-table tr {
     @apply even:bg-dark-charcoal-06;
   }
 
-  .table th {
+  .source-table th {
     @apply first:rounded-ss-sm last:rounded-se-sm;
   }
 
-  .table tr:last-child td {
+  .source-table tr:last-child td {
     @apply border-b first:rounded-es-sm last:rounded-ee-sm;
   }
+
+  .mobile-source-table {
+    @apply md:hidden;
+  }
+
+  .mobile-source-table article {
+    @apply grid border-l border-r border-dark-charcoal-20 p-4 sm:grid-cols-4 md:hidden;
```

Comment:
Does this need its own `md:hidden` if it's nested inside `mobile-source-table`, which already has that?

```suggestion
    @apply grid border-l border-r border-dark-charcoal-20 p-4 sm:grid-cols-4;
```